client/jetpack-cloud/sections/partner-portal/mixins.scss

```diff
@@ -0,0 +1,31 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+@mixin licensing-portal-bottom-action-bar() {
+	position: fixed;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	padding: 0.875rem 0.875rem 1.5rem;
+	background: white;
+	box-shadow: 0 -1px 2px rgba( 0, 0, 0, 0.12 );
+	z-index: 20;
+
+	.button {
+		width: 100%;
+		padding: 8px 24px;
+		font-size: 1rem;
+		white-space: nowrap;
+	}
+
+	@include breakpoint-deprecated( '>660px' ) {
```

Comment:
I am curious why we still put `breakpoint-deprecated` through. Is there any limitation on using Gutenberg's breakpoints?

I am really just curious to understand why and when would the use of `breakpoint-deprecated` be needed.

Reply:
[I asked earlier today](https://github.com/Automattic/wp-calypso/pull/61372#discussion_r812846912) in another PR 😄 

**André:**
> I'm guessing the choice of using `breakpoint-deprecated( '>660px' )` is because other components on the page does it, which makes all components react at the same time; but we could use `@include break-small`
> 
> _Notice: `break-small` breaks at 600px so there's a 60px difference between the breakpoints_

**Atanas:**

> The key point is that the sidebar hides at 660 which completely changes the layout otherwise I would've avoided the deprecated breakpoint.